notice/store.go

```diff
@@ -51,6 +60,24 @@ func (s *Store) FindAllPolicyNotices(ctx context.Context, policyID string) ([]No
 			Details: "To receive alerts for this configuration, assign this escalation policy to its relevant service.",
 		})
 	}
+	type PolicyDuration struct {
+		DelaySum int
+	}
+	cfg := config.FromContext(ctx)
+	var policyDuration PolicyDuration
+	err = s.findPolicyDurationMinutes.QueryRowContext(ctx, policyID).Scan(&policyDuration.DelaySum)
+	if err != nil {
+		return nil, err
+	}
+
+	autoCloseDays := cfg.Maintenance.AlertAutoCloseDays
+
+	if autoCloseDays > 0 && policyDuration.DelaySum/(24*60) >= autoCloseDays {
+		notices = append(notices, Notice{
+			Message: "Auto Closure of uncknowledged/stale alerts",
+			Details: "Alerts using this policy will be automatically closed after " + strconv.Itoa(autoCloseDays) + " days.",
+		})
+	}
```

Comment:
We can simplify this a bit, the struct is unnecessary. Also, use `Sprintf` instead of manually concating with `Itoa`
```suggestion
	var mins int
	err = s.findPolicyDurationMinutes.QueryRowContext(ctx, policyID).Scan(&mins)
	if err != nil {
		return nil, err
	}

	cfg := config.FromContext(ctx)
	days := cfg.Maintenance.AlertAutoCloseDays
	if days > 0 && mins/(24*60) >= days {
		notices = append(notices, Notice{
			Message: "Auto-closure of unacknowledged alerts",
			Details: fmt.Sprintf("Alerts using this policy will be automatically closed after %d day(s).", days),
		})
	}
```